docs/use/dapp-staking/for-stakers/index.md

```diff
@@ -0,0 +1,111 @@
+import Figure from "/src/components/figure"
+
+# For Stakers
+
+## Overview
+
+Do you love to stake? Or do you want to support your favorite project on Astar Network? 
+
+Discover [Astar dApp Staking](/docs/learn/build2earn.md), a unique mechanism that allows communities to support their favorite teams by staking ASTR or SDN with them. When staking ASTR on a dApp, users not only support the development of innovative apps but also receive staking rewards from inflation.
+
+You can educate yourself more by reading the technical documentation about [dApp Staking V3](/docs/learn/dapp-staking-v3/).
```

Comment:
This is now correct link, but in PR https://github.com/AstarNetwork/astar-docs/pull/573 it will be changed to /docs/learn/dapp-staking (without version)